### PR TITLE
Lowers the metabolization rate of nooartrium to 0.2 (while reviving someone)

### PR DESCRIPTION
**Pull request by RikuTheKiller**
## About The Pull Request

I noticed that Nooartrium wasn't multiplying it's metabolism while reviving someone by REM whatsoever.
This, from what I can tell, results in nooartrium having a metabolism of 0.7, which for a massively difficult inverse wittel reagent doesn't seem very justified.

I also noticed the obscene metabolism while trying to use the chemical on a local server. It was blatantly unusable. (You can only inject 19.9u of Nooartrium at most and it rapidly damages your heart.)

This PR changes the weird value of metabolism from 0.35 to 0.2 and also multiplies it by REM in the code. This should result in a 0.2 metabolism while reviving someone instead of 0.7.

## Why It's Good For The Game

Doing all that only to get a chemical that metabolizes this quickly is plain obscene. (I've never seen this chemical being used anyways, this just might solve that.)

It's overdose is also rather low and it deals massive heart damage so I don't get the issue here. If you get a tier 3 cyberheart, wittel and make the complicated ingredients required for penthrite, you should be able to use it to great effect.

You can already revive yourself from the dead in numerous ways, especially strange reagent synthflesh smoke grenades. (Which are a pain to set up, but less of a pain than nooartrium is.) You could also just use one of the many antag revivals. Nooartrium doesn't heal you, either. Oh, right, it also slows you down a lot if your health is down in the dirt.

## Changelog

:cl:
balance: Nooartrium metabolizes much slower now.
/:cl:

### CODE REVIEW

**Comment by Xander3359** [comment]
Having tested this in-game, I do think it could do with some huge buffs. The fact that it's super hard to get should make it that it lasts quite a while.

**Comment by RikuTheKiller** [comment]
> Having tested this in-game, I do think it could do with some huge buffs. The fact that it's super hard to get should make it that it lasts quite a while.

then again it does revive you from the dead so 0.4 is fine i think
0.7, though, isnt

**Comment by RikuTheKiller** [comment]
actually, screw it, changing it to 0.3
50 seconds of not death is barely anything, you can achieve the same effect as this by using a strange reagent grenade which also, if set up correctly, heals you to full
except the grenade doesnt need wittel and whatever the fuck else

theres also ethreals who can revive from the dead

the reagent also massively slows you down depending on health, so like...

**Comment by Xander3359** [comment]
I do think this chem needs a niche carved out for it, I believe it should last quite a while, because it's really only supposed to last enough time for whoever's banged up corpse to get to medbay. Considering you can inject this in someone who has no blood level and 300+ damage, give them 1-2 minutes to get to medbay before the drug runs out and they drop dead again. I can understand this chem being balanced if it were easier to get, but at the moment it requires a fuck you amount of effort to make even just small amounts and I think a chemist who works really hard to get a minorly useful chem that doesn't even work in its own niche is quite disappointing.

**Comment by RikuTheKiller** [comment]
> 

You went to lavaland, found a wittel geyser, plumbed it up, linked it to a receiver/took a sample from it, made pentaerythritol + nitric acid and you know what to do to invert it.

In it's current state the only things you'll notice when using it compared to just using medicines to not die are:
1. It slows you down a fuck ton.
2. It metabolizes at lightspeed.
3. If it's not pure (you can't get it pure atm due to #71356) it deals even more heart damage than usual and causes constant brute damage, meaning it's unusable anyways.
4. If you kill your liver before taking it you can avoid the OD and inject an infinite amount into yourself if you feel like making enough of it due to one of the bugs that #71367 is aimed at fixing. (For which I didn't make an issue report.)

File: code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm [hunk 1]
```diff
@@ -518,7 +518,7 @@ Basically, we fill the time between now and 2s from now with hands based off the
 	var/obj/item/organ/internal/heart/heart = owner.getorganslot(ORGAN_SLOT_HEART)
 	if(!heart || heart.organ_flags & ORGAN_FAILING)
 		return ..()
-	metabolization_rate = 0.35
+	metabolization_rate = 0.2 * REM //Previously 0.35, which without REM multiplication amounted to 0.7 which was WAY, WAY TOO MUCH for a chemical this hard to get
```
Comment (Mothblocks) [inline code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm:521]: No historical comments, they are not useful

```suggestion
	metabolization_rate = 0.2 * REM
```

Comment (RikuTheKiller) [reply code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm]: Uhhh kay guess no fixing this then

**Pull request merged**